packages/kbn-apm-synthtrace/README.md

```diff
@@ -109,12 +110,22 @@ node scripts/synthtrace simple_trace.ts --target=http://admin:changeme@localhost
 
 The script will try to automatically find bootstrapped APM indices. **If these indices do not exist, the script will exit with an error. It will not bootstrap the indices itself.**
 
+### Understanding Scenario Files
+
+Scenario files accept 3 arguments, 2 of them optional and 1 mandatory
+
+| Arguments   | Type      | Description                                                                                                                                          |
+|-------------|:----------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `generate`  | mandatory | This is the main function responsible for returning the events which will be indexed                                                                 |
+| `bootstra`  | optional  | In case some setup needs to be done, before the data is generated, this function provides access to all available ES Clients to play with            |
```

Comment:
Is this a typo? Shouldn't be `bootstrap`?

```suggestion
| `bootstrap`  | optional  | In case some setup needs to be done, before the data is generated, this function provides access to all available ES Clients to play with            |
```